docs/content/docs/data-structures/tree.md

```diff
@@ -5,15 +5,20 @@ menuPosition: 6
 
 ## Introduction
 
-The `SharedTree` distributed data structure (DDS), available in Fluid Framework 2.0 preview and later, is used to store most or all of your application's shared data in a hierarchical structure.
+The `SharedTree` distributed data structure (DDS), available in Fluid Framework 2 and later, is used to store most or all of your application's shared data in a hierarchical structure.
 
-A `SharedTree` object has the following characteristics:
+A `SharedTree` has the following characteristics:
 
+-   It is accessed through a `TreeView` object that exposes all the functionality for reading and editing data within a `SharedTree`.
 -   It has a root and can have several types of internal (i.e., non-leaf) nodes and several types of leaf nodes.
 -   Although there are some exceptions, for the most part, each type of node closely mirrors a familiar JavaScript datatype, such as object, map, array, boolean, number, string, and null.
 -   Again, with exceptions, your code accesses nodes with the syntax of JavaScript and TypeScript, such as dot notation, property assignment, and array indexes.
--   Besides being of the type `SharedTree`, the object will also conform to a schema that your code creates so it has an application-specific strong typing too.
--   The various types of internal nodes can be nested in one another (subject to the constraints of the schema).
+-   A `TreeView` will conform to a schema that your code creates so it has application-specific strong typing.
+-   The various types of internal nodes can be nested (subject to the constraints of the schema).
+
+### TreeView object
+
+The `TreeView` object is accessed via the Fluid Framework `IFluidContainer` object (container). The `TreeView` contains a `root` node that provides access to the complete `SharedTree`. The `TreeView` also provides access to tree-level events.
```

Comment:
Is this a bit oversimplified? I would've thought from IFluidContainer one gets the SharedTree, and then `schematize()`s it into the TreeView? But might be wrong.